services/server/src/sourcify-chains-default.json

```
@@ -2272,5 +2272,14 @@
         "url": "https://scan-testnet.exsat.network"
       }
     }
+  },
+  "5115": {
+    "sourcifyName": "Citrea Testnet",
+    "supported": true,
+    "fetchContractCreationTxUsing": {
+      "blockscoutScrape": {
```

I see it's a Blockscout V2 instance so let'it should be this
```suggestion
      "blockscoutApi": {
```

---

Done, thanks!